### ISSUE TITLE
Progress halted?

### ISSUE THREAD

**Issue by KaffeMyers** [EnhancedAthlete/bbPress-Invision-v4-Converter#1]
Hi!
I just stumbled upon this looking for ways to get away from IPB4. I see it says it's a work in progress, but there has been no updates in over 2 years. Is the project dead?

I will give it a spin regardless, see how it turns out. :]

**Comment by BrianHenryIE** [comment]
Yeah, we successfully used it and didn't need to continue developing it! I think I documented pretty clearly the limits and what its missing that people might want. 

Give it a run on a staging site and let me know how it works out for you. I think most work beyond what I did needs identifying additional WordPress and bbPress plugins and then mapping the IPB metadata as appropriate. 

There's [a thread on the bbPress forums discussing it](https://bbpress.org/forums/topic/dev-invision-v4-converter-sql-field_map/) -- seems maybe forum titles don't transfer, I can't recall now did I get that working or did I just manually do this ~15 forum titles I needed.

**Comment by KaffeMyers** [comment]
Tried it out, and at first glance things look good enough to get the ball rolling (which means "great job, thank you a lot for the effort making this!").

Only immediate problem I find is that logins doesn't seem to work ("Wrong credentials"). I don't know how it would affect anything, but I did an sql dump, imported into a temporary table and did the import from there (as I had problem getting the info I needed to do it straight from the source at the time).

Now, I'm not saying this to get you to do anything, but if you immediately have an idea why the login isn't working, I'd appreciate the input. And just so you don't have to ask, the plugin is active, as mentioned as a requirement for this to work in the readme.

Once again, thanks for this!

**Comment by BrianHenryIE** [comment]
Authentication should definitely work, it's nifty how it's done (bbPress itself no real original work) – it stores the old password in user meta, creates the user (if absent) in WordPress with a random password, then hooks late into WordPress's password check and implements the IPB password algorithm, then if it succeeds saves the inputted password as the user's real WordPress password. As you know, that's why the plugin needs to remain active.

[InvisionV4.php#L1472-L1482](https://github.com/EnhancedAthlete/bbPress-Invision-v4-Converter/blob/6c0249e8fbba142f18fcf596c85e2b5af5b2a18b/InvisionV4.php#L1472-L1482) is the code. It refers to Invision's own code `Member.php:2507`. Could you check yours as see does it have something like `crypt( $password, '$2a$13$' . $pass_array['salt'] ) );`

I don't think you copying the db would be a problem here.
